CLDR-18940 Update release page (#5115)
unicode-org/cldr: docs/site/downloads/cldr-48.md

### Changes

> Some of the most significant changes in this release are:

Reviewer: I'm just trying to avoid the readers' reaction of "well, why don't you tell me all of the significant changes?", but OK.

### Numbers
* [Plural rules syntax](https://www.unicode.org/reports/tr35/dev/tr35-numbers.html#plural-rules-syntax) Added substantial clarifications and new examples.
The order of execution is also clearly specified.
* [Compact Number Formats](https://www.unicode.org/reports/tr35/dev/tr35-numbers.html#compact-number-formats) Specified the mechanism for formatting compact numbers more precisely.
* [Rational Numbers](https://www.unicode.org/reports/tr35/dev/tr35-numbers.html#rational-numbers) Added support for formatting fractions like 5½.

### Units of Measurement
* [Unit Syntax](https://www.unicode.org/reports/tr35/dev/tr35-general.html#unit-syntax) Simplified the EBNF `product_unit` and added an additional well-formedness constraint for mixed units.
* [Unit Identifier Normalization](https://www.unicode.org/reports/tr35/dev/tr35-general.html) Modified the normalization process
* [Mixed Units](https://www.unicode.org/reports/tr35/dev/tr35-general.html#mixed-units) Modified the guidance for handling precision.

### MessageFormat

> * Syntax and data model errors now must be prioritized over other errors <!-- ([\#1011](https://github.com/unicode-org/message-format-wg/pull/1011)) -->

Reviewer: ```suggestion
* Syntax and data model errors must now be prioritized over other errors. <!-- ([\#1011](https://github.com/unicode-org/message-format-wg/pull/1011)) -->
```

> * The Default Bidi Strategy is now required and default <!-- <!-- ([\#1066](https://github.com/unicode-org/message-format-wg/pull/1066)) -->

Reviewer: ```suggestion
* The Default Bidi Strategy is now required and default. <!-- <!-- ([\#1066](https://github.com/unicode-org/message-format-wg/pull/1066)) -->
```

* The `:offset` function (previously named `:math`) is now available as Stable <!-- ([\#1073](https://github.com/unicode-org/message-format-wg/pull/1073)) -->
* The `:datetime`, `:date`, and `:time` functions are updated to build on top of semantic skeletons <!-- ([\#1078](https://github.com/unicode-org/message-format-wg/pull/1078), [\#1083](https://github.com/unicode-org/message-format-wg/pull/1083)) -->

> * `:percent` is added as a new Draft function <!-- ([\#1094](https://github.com/unicode-org/message-format-wg/pull/1094)) -->

Reviewer: ```suggestion
* `:percent` is added as a new Draft function. <!-- ([\#1094](https://github.com/unicode-org/message-format-wg/pull/1094)) -->
```

## Specification Changes

> The following are some of the most significant changes to the specification (LDML).

Reviewer 2: I thought the modifications is a list of the significant changes? Where would any significant changes be that aren't mentioned in the release page?

Maybe if we think there are things in the modifications of the spec but not on the release page we should say something along the lines of 'The following are some of the most significant changes to the specification (LDML). The [modifications] section contains a list of all substantiative changes.' or something similar?

Author: > it up front

Not sure what the "it" is.

Reviewer: I think this sounds subjective. I would prefer then "The following are some of the changes..." if we want to hedge.

Author: See above comment.

The description of how to format names for locale IDs has been extended and clarified with new attributes added for better presentation in menus and other environments `menu="core"`, `menu="extension"`, `scope="core"`. [See Display Name Elements](https://www.unicode.org/reports/tr35/dev/tr35-general.html#display-name-elements) for details.
- Recent or upcoming currency names were added (XCG, ZWG).
- To match ISO, added translations for the region Sark (CQ).
- There are now combination formats for _relative_ dates + times, such as “tomorrow _at_ 12:30”.
In some languages the use of a relative date such as “tomorrow” or “2 days ago” requires a _different_ combining pattern than for a fixed date like “March 20”.
A new “relative” variant is introduced to allow for those languages.
- Added some additional flexible date formats (aka `availableFormats`). 
- Many locales had seldom-used short timezone abbreviations (such as EST) removed, or moved to sublocales that use them.
- The currency-number formats for `alphaNextToNumber`, `noCurrency`, and compact currency formats are now generated from other data for consistency.

> The `alphaNextToNumber` patterns allow for adding a space between letter currency symbols and numbers. For example, "USD 123" vs "US$123".

Reviewer: ```suggestion
The `alphaNextToNumber` patterns allow for adding a space between letter currency symbols and numbers, for example, "USD 123" vs "US$123".
```

#### supplementalData
- `era` — the range of `code` values now allows two letters before the first hyphen.
- `languageData` — the `territories` attribute [`supplementalData.xml`](https://github.com/unicode-org/cldr/blob/main/common/supplemental/supplementalData.xml) was deprecated and data using it removed.

> The definition was unclear, and prone to misunderstanding — the more detailed data is in `territoryInfo`. [CLDR-5708][]

Reviewer: ```suggestion
The definition was unclear and prone to misunderstanding — the more detailed data is in `territoryInfo`. [CLDR-5708][]
```

- `usesMetazone` — adds two new attributes `stdOffset` and `dstOffset` so that implementations can use either "vanguard" or "rearguard" TZDB data sources.
- `numberingSystem` — Unicode 17 data was added.

#### ldmlBCP47
- `type` — adds a new attribute `region`, for determining the region from short timezone IDs when not derivable from the first two characters.
- `keyboard3@conformsTo` — is updated to allow "48".
- `hc` — adds values `c12` and `c24` as Technical Preview. Also see the note about `h24` in "V49 advance warnings". [CLDR-18894][]

### BCP47 Data Changes

For a full listing, see [BCP47 Delta].

- `nu-tols` numbering system for Tolong Siki digits
- One additional zone: 	America/Coyhaique = tz-clcxq

> - Region attributes in timezones, for determining regions where it can't be derived from the short TZID 

Reviewer 2: ```suggestion
- Region attributes in timezones for determining regions where it can't be derived from the short TZID 
```

Reviewer: Should have a period too, right?

- Number patterns that did not have a specific numberSystem (such as `latn` or `arab`) had been deprecated for many releases, and were finally removed.
- Additionally, language and territory data in `languageData` and `territoryInfo` data received significant updates to improve accuracy and maintainability [CLDR-18087]
- The likely language for Belarus changed to Russian [CLDR-14479]

> - [Using Time Zone Names](https://www.unicode.org/reports/tr35/dev/tr35-dates.html#using-time-zone-names) Removed the "specific location format", and modified the fallback behavior of 'z'.

Reviewer: ```suggestion
- [Using Time Zone Names](https://www.unicode.org/reports/tr35/dev/tr35-dates.html#using-time-zone-names) Removed the "specific location format" and modified the fallback behavior of 'z'.
```

> - [Unit Identifier Normalization](https://www.unicode.org/reports/tr35/dev/#tr35-general.html) Modified the normalization process

Reviewer: ```suggestion
- [Unit Identifier Normalization](https://www.unicode.org/reports/tr35/dev/#tr35-general.html) Modified the normalization process.
```

### V49 advance warnings

The following changes are planned for CLDR 49. Please plan accordingly to avoid disruption.
- H24 will be deprecated. If it is encountered, it will have H23 behavior. There is no known intentional usage of H24. If you have a current need for H24 instead of H23, please comment on [CLDR-18303][].

> - The default week numbering changes to ISO instead being based on the calendar week starting in CLDR 48. The calendar week will be more clearly targeted at matching usage in displayed month calendars. [CLDR-18275][ ]

Reviewer: ```suggestion
- The default week numbering system changes to follow ISO, instead of being based on the calendar week, starting in CLDR 48. The calendar week will be more clearly targeted at matching usage in displayed month calendars. [CLDR-18275][ ]
```

> - The default week numbering system changes to follow ISO, instead of being based on the calendar week, starting in CLDR 48. The calendar week will be more clearly targeted at matching usage in displayed month calendars. [CLDR-18275][]

Reviewer 2: ```suggestion
- As of CLDR 49, the default week numbering system will change to follow ISO (where weeks are numbered based on Thursday), instead of being based on the start of the calendar week. The calendar week will be more clearly targeted at matching usage in displayed month calendars. [CLDR-18275][]
```
